lib/themes/dosomething/paraneue_dosomething/js/campaign/HotModule.js

```diff
@@ -103,6 +103,13 @@ const createQuantityArray = function(dates, dateLabels, quantities) {
 };
 
 /*
+ * Checks to see if we only have one point of data.
+ */
+const hasOnePoint = function(data) {
+  return (JSON.parse(data.quantities) && JSON.parse(data.quantities).length === 1) ? true : false;
```

Comment:
I'd recommend running `JSON.parse()` on the `progressData` back when you load it from the settings object, so you don't have to re-parse it each time you want to do something with it.